src/SessionManager/ProgramSettings.cc

```diff
@@ -38,41 +40,73 @@ ProgramSettings::ProgramSettings(int argc, char** argv) {
+        ("exit_after", "number of seconds to stay alive after last session exits", cxxopts::value<int>(), "<sec>")
+        ("init_exit_after", "number of seconds to stay alive at start if no clients connect", cxxopts::value<int>(), "<sec>")
+        ("files", "files to load", cxxopts::value<vector<string>>(positional_arguments));
 
     options.add_options("Deprecated and debug")
-        ("debug_no_auth", "Accept all incoming WebSocket connections on the specified port (not secure; use with caution!)", cxxopts::value<bool>())
-        ("threads", "[Deprecated] No longer supported.", cxxopts::value<int>(), "<threads>")
-        ("base", "[Deprecated] Set starting folder for data files. Use the positional parameter instead.", cxxopts::value<string>(), "<dir>")
-        ("root", "[Deprecated] Use 'top_level_folder' instead.", cxxopts::value<string>(), "<dir>");
+        ("debug_no_auth", "accept all incoming WebSocket connections on the specified port (not secure; use with caution!)", cxxopts::value<bool>())
+        ("threads", "[deprecated] no longer supported", cxxopts::value<int>(), "<threads>")
+        ("base", "[deprecated] set starting folder for data files (use the positional parameter instead)", cxxopts::value<string>(), "<dir>")
+        ("root", "[deprecated] use 'top_level_folder' instead", cxxopts::value<string>(), "<dir>");
     // clang-format on
 
-    options.positional_help("<file or folder to open> (current directory is the default)");
+    options.positional_help("<file or folder to open>");
     options.parse_positional("files");
     auto result = options.parse(argc, argv);
 
+    std::string log_levels(R"(
+ 0   off
+ 1   critical
+ 2   error
+ 3   warning
+ 4   info
+ 5   debug
+ 6   trace)");
+
+    std::string extra = fmt::format(R"(
+By default the CARTA backend uses the current directory as the starting data 
+folder, and uses the root of the filesystem (/) as the top-level data folder. If 
+a custom top-level folder is set, the backend will be restricted from accessing 
+files outside this directory.
+
+Frontend files are served from '../share/carta/frontend' (relative to the 
+location of the backend executable). By default the backend listens for HTTP and 
+WebSocket connections on all available interfaces, and automatically selects the 
+first available port starting from {}.  On startup the backend prints out a URL 
+which can be used to launch the frontend, and tries to open this URL in the 
+default browser.
+
+The gRPC service is disabled unless a gRPC port is set. By default the number of 
+OpenMP threads is automatically set to the detected number of logical cores.
+
+Logs are written both to the terminal and to a log file, '.carta/log/carta.log' 
+in the user's home directory. Possible log levels are:{}
+
+Options are provided to shut the backend down automatically if it is idle.
+)",
+        DEFAULT_SOCKET_PORT, log_levels);
```

Comment:
isn't that list of log levels going to break your wrapping?

Reply:
No, they're all very short lines. It's in a separate variable so that the wrapping doesn't break it.